isolate_example/lib/page_two.dart

```diff
@@ -157,17 +164,28 @@ class RunningList extends StatelessWidget {
 
     return DecoratedBox(
       decoration: BoxDecoration(
-        color: (controller.running == true && controller.paused == false)
-            ? Colors.lightGreenAccent
-            : Colors.deepOrangeAccent,
+        color: Colors.grey[200],
       ),
       child: new ListView.builder(
         itemCount: sums.length,
         itemBuilder: (context, index) {
-          return ListTile(
-            title: Text((sums.length - index).toString() +
-                '. ' +
-                sums[index].toString()),
+          return Column(
+            children: <Widget>[
+              Card(
+                child: ListTile(
+                  leading: Text((sums.length - index).toString() + '. '),
```

Comment:
Have we talked about Dart's string interpolation yet?

https://dart.dev/guides/language/language-tour#strings

You can rewrite this as:

```
Text(${sums.length - index}.')
```

The one on line 177 can also be done with interpolation.